programs/fileio.c

```diff
@@ -1425,6 +1430,18 @@ static int FIO_compressFilename_dstFile(FIO_prefs_t* const prefs,
     return result;
 }
 
+static const char *compressedFileExtensions[] = {
```

Comment:
Can you distinguish this list from the `suffixList` added in #1848? I see that this always includes all compression algorithms, even when zstd isn't built with support for them, whereas the other only has extensions zstd itself can decompress. It seems reasonable that the behavior you're adding shouldn't depend on zstd's build settings in that way, so maybe this should be a separate list. If so though, can you add a comment describing the difference between the two and how they're used?

Reply:
Agreed. Addressed in latest commit.